app/static/js/zseries_recipe.js

```diff
@@ -131,6 +135,7 @@ function update_recipe(recipe_id){
         var recipe = {};
         recipe.id = recipe_id
         recipe.steps = table.getData();
+        // convert temperatures to F
```

Comment:
?

Reply:
Yeah that comment isn't needed, can delete it. Since tabulator is just "presenting" temps in °C when requested the data table still contains the °F value. No need to convert here.